design-documents/graph-ql/coverage/product-reviews.graphqls

```diff
@@ -11,30 +11,33 @@ type ProductInterface {
     reviews(
         pageSize: Int = 20 @doc(description: "Specifies the maximum number of results to return at once."),
         currentPage: Int = 1 @doc(description: "Specifies which page of results to return."),
-    ): ProductReviews!
+        sort: ReviewSortInput @doc(description: "Specifies which field to sort on, and whether to return the results in ascending or descending order.")
+    ): ProductReviews! @doc(description: "ProductReview returns an array containing approved product reviews")
 }
 
 type ProductReviews {
     items: [ProductReview]! @doc(description: "An array of product reviews.")
     page_info: SearchResultPageInfo! @doc(description: "Metadata for pagination rendering.")
+    total_count: Int @doc(description: "The number of product reviews returned")
 }
 
 type ProductReview @doc(description: "Details of a product review") {
+    review_id: Int! @doc(description: "The ID assigned to the review")
     product: ProductInterface! @doc(description: "Contains details about the reviewed product")
-    summary: String! @doc(description: "The review summary (a.k.a title")
-    text: String! @doc(description: "The review text.")
+    title: String! @doc(description: "The review title")
```

Comment:
Based on https://github.com/magento/architecture/pull/343#discussion_r461735711 
let's revert this change.